rotkehlchen/tests/unit/globaldb/test_globaldb_cache.py

```diff
@@ -238,6 +240,37 @@ def test_velodrome_cache(optimism_inquirer):
     assert mock_notify.call_args_list == [make_call_object(CPT_VELODROME, ChainID.OPTIMISM, 0, 0)]
 
 
+class MockEvmContract:
+    def call(self, *args, **kwargs):
+        if kwargs['arguments'][1] == 0:
+            return [{
+                0: '0x3241738149B24C9164dA14Fa2040159FFC6Dd237',
```

Comment:
can you add a small comment to this class explaining what it does and the test where it is used